### PR TITLE
Feat/ZMB-25 Propose book form view

### PR DESCRIPTION
**Pull request by mariacuadroszm**
Create the components:
1. Form label
2. Main header
3. Propose book form (view)

All the buttons are styleless, they'll be added after @lesliefernandez10 update the button component. The final styles are pending until the designer finish them.

### CODE REVIEW

File: src/router/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { createRouter, createWebHistory } from "vue-router";
 import HomeView from "../views/HomeView.vue";
+import ProposeBookForm from "../viewS/ProposeBookForm.vue";
```
Comment (josezumaque) [inline src/router/index.js:3]: Typo error: `viewS`

File: src/views/ProposeBookForm.vue [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+      synopsis: "",
+    };
+  },
+  computed: {
+    titleUpperCase() {
+      return this.title.toUpperCase();
+    },
+    authorUpperCase() {
+      return this.author.toUpperCase();
+    },
```
Comment (josezumaque) [inline src/views/ProposeBookForm.vue:109]: Please remove this code if you are not using it: `titleUpperCase()` and `authorUpperCase()`.

**Review by josezumaque — CHANGES_REQUESTED** [review]

File: src/views/ProposeBookForm.vue [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+        <FormLabel id="title">
+          <template v-slot:labelDescription>Title</template>
+          <template v-slot:labelInput>
+            <input
+              type="text"
+              id="title"
+              placeholder="Type book's title"
+              maxlength="150"
+              required
+              class="label__input label__input-caps"
```
[the remark this answers is not in the document]
Comment (mariacuadroszm) [reply src/views/ProposeBookForm.vue:20]: Ok, I'll correct it

File: src/router/index.js (hunk 1 above)
Comment (mariacuadroszm) [reply src/router/index.js:3]: Ok, I'll correct it

File: src/views/ProposeBookForm.vue (hunk 2 above)
Comment (mariacuadroszm) [reply src/views/ProposeBookForm.vue:109]: I created this methods to make sure the input info I'll be sending to the API is in caps when we do the integration with backend.

**Review by josezumaque — APPROVED** [review]

**Pull request merged**